src/components/FilterPane/FilterPane.jsx

```diff
@@ -197,12 +212,24 @@ const FilterPane = ({ mermaidUserData }) => {
             </Box>
           )}
         >
-          {countries.map((country) => (
+          {remainingDisplayedCountries.length > 0 ? (
+            <StyledListSubheader>Countries based on current filters</StyledListSubheader>
+          ) : null}
+          {displayedCountries.map((country) => (
             <StyledMenuItem key={country} value={country}>
               <input type="checkbox" checked={selectedCountries.includes(country)} readOnly />
               {country}
             </StyledMenuItem>
           ))}
+          {remainingDisplayedCountries.length > 0 ? (
+            <StyledListSubheader>Other countries</StyledListSubheader>
+          ) : null}
+          {remainingDisplayedCountries.map((country) => (
+            <StyledMenuItem key={`other-${country}`} value={country}>
```

Comment:
Can you be more explicit with the key values here? I think something other than "key={country}" and  "key={`other-${country}`}" would be clearer